src/fidesops/ops/models/privacy_request.py

```diff
@@ -104,6 +107,7 @@ class Config:
 class PrivacyRequestStatus(str, EnumType):
     """Enum for privacy request statuses, reflecting where they are in the Privacy Request Lifecycle"""
 
+    identity_unverified = "identity_unverified"
```

Comment:
I'm trying to think whether `identity_unverified` will always be set or only be set for requests where identity verification is turned on. This is more of a future convo, since I was doing some thinking here around edge cases- https://github.com/ethyca/fidesops/issues/1009#issuecomment-1221012566

Reply:
Yeah, different ways to organize.  I was viewing it as a state that was only valid when we had identity verification on. 
  
My work in #1010 (no PR yet)  has a workflow that sets initial status as `identity_unverified` when identify verification is "on", otherwise, it sets the initial status to `pending`.  If the identity is verified, then I change the status from `identity_unverified` to `pending`.

Reply:
cool, so depending on confirmation from product on https://github.com/ethyca/fidesops/issues/1009#issuecomment-1221012566, your workflow sounds good to me!